### PR TITLE
included subroutines for hybrid-lio

### PR DESCRIPTION
**Pull request by Nick-Chaos**
Included 2 subroutines for make QMMM calculations using hybrid program.

### CODE REVIEW

File: lioamber/init_lio.f90 [hunk 1]
```diff
@@ -428,3 +428,33 @@ subroutine init_lioamber_ehren(natomin, Izin, nclatom, charge, basis_i         &
...
+!%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%!
+!%% INIT_LIO_HYBRID  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%!
+! Subroutine init_lio_hybrid performs Lio initialization when called from      !
+! Hybrid software package, in order to conduct a hybrid QM/MM calculation.     !
+!%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%!
+subroutine init_lio_hybrid(hyb_natom, mm_natom, charge, iza)
+    implicit none
+    integer, intent(in) :: hyb_natom !number of total atoms
+    integer, intent(in) :: mm_natom  !number of MM atoms
+    integer             :: dummy
```
Comment (manuelF) [inline lioamber/init_lio.f90]: Why do you have a dummy parameter?

File: lioamber/SCF_exter.f90 [hunk 2]
```diff
@@ -175,3 +175,82 @@ subroutine SCF_gro(E, qmcoords, clcoords, clcharge, nsolin)
...
+        fdummy(j,i)=fa(j,i)
+      end do
+      do i=1, mm_natom
+        fdummy(j,hyb_natom+i)=fmm(j,i)
+      end do
+    end do
+
+    return
+end  subroutine SCF_hyb
+
```
Comment (manuelF) [inline lioamber/SCF_exter.f90]: Too many newlines at end.

File: lioamber/init_lio.f90 (hunk 1 above)
Comment (Nick-Chaos) [reply lioamber/init_lio.f90]: it is same case as init_lio_gromacs and init_lio_amber subroutines, all need to call read_options with a dummy parameter

File: lioamber/SCF_exter.f90 (hunk 2 above)
Comment (Nick-Chaos) [reply lioamber/SCF_exter.f90]: my bad, sorry

**Comment by Nick-Chaos** [comment]
No in lio, all these routines are made for use lio as external library in "hybrid" program, so test and other stuff will be included in a future official version of hybrid.

**Comment by ramirezfranciscof** [comment]
I think we still should include some sort of test for this sections in our own code. Maybe some program that specifically uses liolib by calling these subroutines with specific parameters and expecting an specific output? Kind of a "systemic" unit test. Not saying it has to be now, but eventually all code should be tested.

**Pull request merged**